src/main/java/club/mher/compass/Compass.java

```diff
@@ -1,43 +1,39 @@
 package club.mher.compass;
 
+import club.mher.compass.command.bw1058.CompassMenuCommand;
+import club.mher.compass.data.IMainConfig;
+import club.mher.compass.data.bw1058.MainConfig;
+import club.mher.compass.listener.bw1058.GameListener;
+import club.mher.compass.listener.MenuListener;
 import club.mher.compass.listener.QuickBuyListener;
+import club.mher.compass.support.Misc;
 import club.mher.compass.support.bstats.Metrics;
-import com.andrei1058.bedwars.api.BedWars;
-import com.andrei1058.bedwars.api.arena.IArena;
-import com.andrei1058.bedwars.api.arena.team.ITeam;
+import club.mher.compass.support.vault.VaultSupport;
 import lombok.Getter;
-import club.mher.compass.command.CompassMenuCommand;
-import club.mher.compass.data.MainConfig;
 import club.mher.compass.data.MessagesData;
-import club.mher.compass.listener.GameListener;
-import club.mher.compass.listener.MenuListener;
-import club.mher.compass.support.vault.VaultSupport;
 import org.bukkit.Bukkit;
 import org.bukkit.plugin.java.JavaPlugin;
 
 import java.io.File;
 import java.util.Arrays;
-import java.util.HashMap;
-import java.util.UUID;
 import java.util.logging.Level;
 
 public class Compass extends JavaPlugin {
 
     @Getter private static Compass instance;
-    @Getter private static BedWars bedWars;
-    @Getter private static MainConfig mainConfig;
-    @Getter private static final HashMap<IArena, HashMap<UUID, ITeam>> trackingArenaMap = new HashMap<>();
     @Getter private static boolean isUsingVaultChat = false, isUsingPapi = false;
     @Getter private static VaultSupport vault;
 
     public static final String VERSION = Bukkit.getServer().getClass().getName().split("\\.")[3];
     public static String PLUGIN_VERSION;
+    public static String HOOK_NAME;
+    public static IMainConfig config;
 
     @Override
     public void onEnable() {
         instance = this;
-        if (!Bukkit.getPluginManager().isPluginEnabled("BedWars1058")) {
-            getLogger().severe("BedWars1058 was not found. Disabling...");
+        if (!Bukkit.getPluginManager().isPluginEnabled("BedWars1058") && !Bukkit.getPluginManager().isPluginEnabled("BedWars2023") ) {
```

Comment:
isn't better an OR instead of an AND here?

Reply:
Its a check if neither plugins are installed. and if thats true the plugin will be disabled.

Reply:
Note that they are both inverted

Reply:
I ask because reading the code if I have bw2023 and not bw1058 in it, the plugin will still be disabled for me because they are not all 2 in it

Reply:
If 1 is installed it will be (!true && !false) -> (true && false) -> (false)
if 2 is installed it will be (!true && !true) -> (false && false) -> (false)
if 0 is installed it will be (!false && !false) -> (true && true) -> (true)

Reply:
ok